docs/live/apply.md

```diff
@@ -40,4 +30,89 @@ Flags:
     
   wait-timeout:
     The threshold for how long to wait for all resources to reconcile before
-    giving up. The default value is 1 minute.
\ No newline at end of file
+    giving up. The default value is 1 minute.
+
+#### `kubectl apply` vs `kpt live apply`
+
+|                     | `kubectl apply`            | `kpt live apply`          |
+|---------------------|----------------------------|---------------------------|
+|Usage                | kubectl apply -f /dir      | kpt live apply /dir       |
+|Applied resource set | Not tracked                | Tracked                   |
+|Prune                | Imperative and error prone | Declarative and reliable  |
+|wait-for-reconcile   | Not supported              | Supported                 |
+
+Usage:
+  In terms of usage, both `kubectl apply` and `kpt live apply` follow similar pattern.
+  The user experience remains unchanged.
+
+Applied resource set(ARS):
```

Comment:
If we are using this terminology, I think we need to describe what it is.

Reply:
Done.